shared/src/main/java/gov/hhs/cdc/trustedintermediary/context/ApplicationContext.java

```diff
@@ -76,15 +101,15 @@ private static void injectIntoField(Field field, boolean skipMissingImplementati
         declaringClassesToTry.add(declaringClass);
         declaringClassesToTry.addAll(Arrays.asList(declaringClass.getInterfaces()));
 
-        Object fieldImplementation = getFieldImplementation(fieldType, skipMissingImplementations);
+        Object fieldImplementation = getFieldImplementation(fieldType);
         if (fieldImplementation == null) {
             return;
         }
 
         Object declaringClassImplementation =
-                getDeclaringClassImplementation(declaringClassesToTry, skipMissingImplementations);
+                getDeclaringClassImplementation(declaringClassesToTry);
         if (declaringClassImplementation == null) {
-            return;
+            System.err.println("No class implementation found");
```

Comment:
We already have an log statement printed on line 165 of this file.  And if that log statement is being printed, this log statement is being printed.  We don't need both log statements.

If we're going to keep this if statement, I would prefer we keep the `return` so we don't throw an unnecessary `NullPointerException`.  Thoughts?

Reply:
I'm ok with that.